src/scenes/Projects/Members/List/ProjectMembersList.tsx

```diff
@@ -87,9 +101,24 @@ export const ProjectMembersList: FC = () => {
             key={item?.id ?? index}
             projectMember={item}
             className={classes.item}
+            onEdit={() =>
+              openUpdateProjectMemberDialog({
+                projectMemberId: item?.id,
+                userId: item?.user.value?.id,
+                userRoles: item?.roles.value,
```

Comment:
This (combined with lines 117-119) seems like it's putting the burden on `UpdateProjectMember` to handle empty values, when really those values would just make it non-functional. It would be better to make all these parameters required in the type variable we're passing to `useDialog` above, so that it matches the typings for `UpdateProjectMember`. Then we should do the proper checking inside this `.map()` to avoid rendering anything at all if `item` is `undefined`. 

We'd still have the problem of needing the `userId`, though. Could `ProjectMemberCard` be a wrapper around `FieldOverviewCard`? It was recently updated to handle `canRead` and `canEdit`, and (among other things) avoid displaying an "Edit" button if `canEdit` is `false`. You can see one example of how its used in the `FilesOverviewCard`.

Reply:
> Then we should do the proper checking inside this .map() to avoid rendering anything at all if item is undefined.
We still need to render to show a loading list, but I did make the props required

> Could ProjectMemberCard be a wrapper around FieldOverviewCard?
I think this is a bigger ask that I'd like to do outside of this PR if we choose to go this route.

Reply:
> We still need to render to show a loading list

Lol, wasn't thinking